oracle-simple.cabal

```diff
@@ -33,9 +33,14 @@ library
   exposed-modules:
     Database.Oracle.Simple
   other-modules:
-    Database.Oracle.Simple.Internal, Database.Oracle.Simple.FromRow, Database.Oracle.Simple.FromField
+      Database.Oracle.Simple.Internal
+    , Database.Oracle.Simple.FromRow
+    , Database.Oracle.Simple.FromField
+    , Database.Oracle.Simple.TableInfo
+    , Database.Oracle.Simple.ToRow
+    , Database.Oracle.Simple.ToField
   build-depends:
-    base < 5, derive-storable, text, mtl
+    base < 5, derive-storable, text, mtl, casing
```

Comment:
Are we using `casing` ?

Reply:
no longer needed when we spin off `HasTableInfo` to another branch. It was needed to derive an oracle table name from the record type